pywhat/identifier.py

```diff
@@ -12,24 +12,42 @@ def __init__(self):
 
     def identify(self, text: str, api=False) -> dict:
         identify_obj = {}
+        magic_numbers = {}
+        file_text = []
+
+        if not api and (os.path.isdir(text) == True or os.path.isfile(text) == True):

```

Comment:
What should the behaviour of the API be _if_ it was given a file path?

Reply:
I have removed this, so if API was given a file path it will work the same. Or should it be done differently?

Reply:
No, it should work like that I believe! Make sure to include tests for if the API was given a file path :)

Reply:
Done!